### ISSUE TITLE
.jks file does not need to be in classpath

### ISSUE THREAD

**Issue by pieterlukasse** [thehyve/cbioportal-docker#28]
you can refer to them as `file:/... `. This means that COPY command for these 2 files could be done **after** the maven step. This would improve/speedup the process when changing any of these files.

**Comment by fedde-s** [comment]
Right, even if the code or the (compile-time) configuration properties may change more often than these files, compiling takes a lot of time and copying these files into the image doesn't.

**Comment by fedde-s** [comment]
I believe that the Keycloak integration docs here have now been out of date for two or three releases, since all configuration except log4j.properties moved to runtime. Fixing that may also address this issue.

**Comment by fedde-s** [comment]
The Catalina context file has been factored out altogether; see #27.

**Comment by pieterlukasse** [comment]
@fedde-s any updates on this one?
